command/agent/template/template.go

```diff
@@ -80,7 +92,12 @@ func (ts *Server) Run(ctx context.Context, incoming chan string, templates []*ct
 	// construct a consul template vault config based the agents vault
 	// configuration
 	var runnerConfig *ctconfig.Config
-	if runnerConfig = newRunnerConfig(ts.config, templates); runnerConfig == nil {
+	var runnerConfigErr error
+	if runnerConfig, runnerConfigErr = newRunnerConfig(ts.config, templates); runnerConfigErr != nil {
+		ts.logger.Error("template server failed to generate runner config", "error", runnerConfigErr)
+		return
+	}
+	if runnerConfig == nil {
```

Comment:
This check might be unneeded since it looks like it's not possible for `newRunnerConfig` to return `nil, nil`.

Reply:
Indeed, thanks! Removed in 6e3dbb9e2